### PR TITLE
Create-djsce-edu-in

### PR DESCRIPTION
**Pull request by SwapneelM**
@philipto I have previously successfully added "www.djsce.ac.in" as the official website of our college to JetBrains/swot
**tl;dr**
Dear Sir, kindly take a minute to look up both domains "www.djsce.ac.in" and "www.djsce.edu.in" on whois domain lookup. You will find both registered with the same details, to the same admin email address belonging to Shri Vile Parle Kelavani Mandal (S.V.K.M.), the trust which runs our institution. I have verified the same already.The reason I need to add this domain is because student email addresses provided are at the "djsce.edu.in" domain while teachers get email addresses at "djsce.ac.in"

Dear Sir, the website is under construction, however they have provided us with student email addresses at the same domain name. We haven't been informed as to exactly when the website will go live. Only Professors are provided email addresses at the domain "www.djsce.ac.in" while student emails are available only on request, at "www.djsce.edu.in"
We have requested multiple times to at least involve students in developing the website so it can be done faster and we have an idea of what's going on however we received no response as to who is in charge of developing the website. 
All I can ask is for you to look up both domains on whois. You will find both registered with the same details, to the same admin email address which is Shri Vile Parle Kelavani Mandal, the trust which runs the institution. I have verified the same. 
I would urge you to merge the pull request as there are a lot of us students with email addresses at the same domain wishing to avail of the JetBrains software but cannot do so.

### CODE REVIEW

**Comment by philipto** [comment]
@SwapneelM Pull request merged. I explained the initial problem in #1894. You can disregard my appeal to reopen request #1894, because I merged this request, and it repeats #1894, so no action from your side is required. Thank you!

**Comment by SwapneelM** [comment]
Appreciate the understanding. Thanks, Sir!

On Tue, Apr 26, 2016 at 2:51 PM, Philip Torchinsky <notifications@github.com
